src/Sonata/Bundle/DemoBundle/DataFixtures/ORM/LoadMediaData.php

```diff
@@ -55,81 +55,59 @@ public function getOrder(): int
+        $this->addMediaToGallery($media, $homepageGallery, false);
 
-        foreach ($paris as $file) {
-            $media = $this->mediaManager->create();
-            $media->setBinaryContent(__DIR__.'/../data/files/hugo-paris/'.$file->getRelativePathname());
-            $media->setEnabled(true);
-            $media->setName('Paris');
-            $media->setDescription('Paris');
-            $media->setAuthorName('Hugo Briand');
-            $media->setCopyright('CC BY-NC-SA 4.0');
-            $media->setCategory($this->getReference('travels_paris_category'));
+        $homepageGallery->setEnabled(true);
+        $homepageGallery->setName('Homepage gallery');
+        $homepageGallery->setDefaultFormat('small');
+        $homepageGallery->setContext('default');
 
-            $this->addReference('sonata-media-'.($i++), $media);
+        $this->galleryManager->update($homepageGallery);
 
-            $this->mediaManager->save($media, 'default', 'sonata.media.provider.image');
-
-            $this->addMediaToGallery($media, $gallery);
-        }
-
-        foreach ($switzerland as $file) {
-            $media = $this->mediaManager->create();
-            $media->setBinaryContent(__DIR__.'/../data/files/sylvain-switzerland/'.$file->getRelativePathname());
-            $media->setEnabled(true);
-            $media->setName('Switzerland');
-            $media->setDescription('Switzerland');
-            $media->setAuthorName('Sylvain Deloux');
-            $media->setCopyright('CC BY-NC-SA 4.0');
-            $media->setCategory($this->getReference('travels_switzerland_category'));
-
-            $this->addReference('sonata-media-'.($i++), $media);
-
-            $this->mediaManager->save($media, 'default', 'sonata.media.provider.image');
-
-            $this->addMediaToGallery($media, $gallery);
-        }
-
-        $gallery->setEnabled(true);
-        $gallery->setName($this->faker->sentence(4));
-        $gallery->setDefaultFormat('small');
-        $gallery->setContext('default');
-
-        $this->galleryManager->update($gallery);
-
-        $this->addReference('media-gallery', $gallery);
+        $this->addReference('media-homepage-gallery', $homepageGallery);
     }
 
-    public function addMediaToGallery(MediaInterface $media, GalleryInterface $gallery): void
+    public function addMediaToGallery(MediaInterface $media, GalleryInterface $gallery, bool $enabled = true): void
     {
         $galleryHasMedia = new GalleryHasMedia();
         $galleryHasMedia->setMedia($media);
         $galleryHasMedia->setPosition(\count($gallery->getGalleryHasMedias()) + 1);
-        $galleryHasMedia->setEnabled(true);
+        $galleryHasMedia->setEnabled($enabled);
 
         $gallery->addGalleryHasMedias($galleryHasMedia);
     }
+
+    protected function createMedia($file, string $name, string $autor, Category $category, bool $enabled = true, string $copyright = 'CC BY-NC-SA 4.0')
```

Comment:
Can this method be private?
```suggestion
    protected function createMedia(string $file, string $name, string $autor, Category $category, bool $enabled = true, string $copyright = 'CC BY-NC-SA 4.0')
```

Reply:
`$file` can be `string` or `File` like `setBinaryContext`. 

IMHO we should keep Api open as much as it is possible in Demo. This will allow user to people to test it and understand how sonata work.

Reply:
It's ok then. Could you please add a dockblock fot this argument?